packages/webapp/src/elections/components/ongoing-election.tsx

```diff
@@ -32,11 +34,12 @@ export const OngoingElection = ({ election }: { election: any }) => {
         isLoading: isLoadingMemberStats,
```

Comment:
`const { data: loggedInUser } = useCurrentMember();` can go away a few lines above this.